cmd/notation/login.go

```diff
@@ -44,13 +44,18 @@ Example - Login using $NOTATION_USERNAME $NOTATION_PASSWORD variables:
 			if err := readPassword(opts); err != nil {
 				return err
 			}
+
+			// check username
+			if opts.Username == "" {
+				return errors.New("username is not set.")
+			}
```

Comment:
If username is not set, then the CLI should prompt instead of an error.
@yizha1 Does `notation` follows the same UX with `oras` and `docker`? or it has a different UX that does not support prompting?

Reply:
Removed this logic.